src/kontrol/kdist/cheatcodes.md

```diff
@@ -907,6 +913,64 @@ Otherwise, throw an error for any other call to the Foundry contract.
       [owise]
 ```
 
+Mock calls
+----------
+
+#### `mockCall` - Mocks all calls to an address where if the call data either strictly or loosely
```

Comment:
IIUC, the convention is to have one line per sentence in Markdown files (@anvacaru please correct me if I'm wrong!).